webui/src/lib/components/repository/changes.jsx

```diff
@@ -153,7 +153,7 @@ export const ChangesTreeContainer = ({results, delimiter, uriNavigator,
                     <div>{changesTreeMessage}</div>
                     <Card>
                         <Card.Header>
-                            <span className="float-start">
+                            <span className="float-start w-100">
```

Comment:
For some reason, `width: 100%` was added globally to `float-start`, which is wrong.
Removed it, and added `w-100` to all the places using it (and not overriding).